weed/server/filer_ui/filer.html

```diff
@@ -288,6 +288,12 @@ <h1>
     function uploadFile(file, i) {
         var url = window.location.href;
         var xhr = new XMLHttpRequest();
+        xhr.onreadystatechange = function() {
+            if (xhr.readyState == XMLHttpRequest.DONE) {
+                var resp = JSON.parse(xhr.responseText);
+                alert(`uploaded file: ${resp.name} with size: ${resp.size}`);
```

Comment:
* I prefer just refreshing the page instead of adding an alert that users need to click away.
* What if there are multiple files uploaded?

Reply:
fixed